From f128f29c3879c55e7cc62dc7b2c6543ce77c4070 Mon Sep 17 00:00:00 2001
From: Wang Jie <dave.wang@rock-chips.com>
Date: Wed, 20 Nov 2019 14:53:46 +0800
Subject: [PATCH] input: sensors: increase sampling frequency.

for android-10.0: increase the sampling frequency to pass the vts test.

Change-Id: I2d68b51e5697a576acebff03b5b1e92b1c90a7d8
Signed-off-by: Wang Jie <dave.wang@rock-chips.com>
---
 drivers/input/sensors/sensor-dev.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/input/sensors/sensor-dev.c b/drivers/input/sensors/sensor-dev.c
index f78493319b58..ba65346c11fb 100644
--- a/drivers/input/sensors/sensor-dev.c
+++ b/drivers/input/sensors/sensor-dev.c
@@ -491,10 +491,10 @@ static int sensor_reset_rate(struct i2c_client *client, int rate)
 	dev_info(&client->dev, "set sensor poll time to %dms\n", rate);
 
 	/* work queue is always slow, we need more quickly to match hal rate */
-	if (sensor->pdata->poll_delay_ms == (rate - 2))
+	if (sensor->pdata->poll_delay_ms == (rate - 4))
 		return 0;
 
-	sensor->pdata->poll_delay_ms = rate - 2;
+	sensor->pdata->poll_delay_ms = rate - 4;
 
 	if (sensor->status_cur == SENSOR_ON) {
 		if (!sensor->pdata->irq_enable) {
-- 
2.35.3

